app/controllers/api/solutions_controller.rb

```diff
@@ -28,7 +28,7 @@ def show
     output = {
       solution: SerializeSolution.(@solution)
     }
-    output[:iterations] = SerializeIterations.(@solution.iterations) if sideload?(:iterations)
+    output[:iterations] = SerializeIterations.(@solution.iterations, sideload: %i[files automated_feedback]) if sideload?(:iterations)
```

Comment:
I've reverted this now, but on any refetch automated_feedback would be missing. Which raises the question whether we need to do any fetching on iterationsList (`/tracks/[track]/exercises/[exercise]/iterations`). I think we don't.

In another PR I'd like to change how this page works, handles data. Because here the data is held inside the query cache, but since we don't do any fetching, this makes little sense. The data inside the cache comes from ruby in the form of `initial_data`. 
We should just hold this initial data in a useState state. We do mutation on deleting iterations, but we can filter frontend data using the response of that. So to me this pattern feels more complex than it should be.